### ISSUE TITLE
MISP Docker-Compose via Portainer

### ISSUE THREAD

**Issue by PandaLyfe** [coolacid/docker-misp#81]
I'm currently using ubuntu 20.04 with docker/docker-compose installed and I'm managing other applications using Portainer. I'm trying to install MISP with Portainer and so far everything appears to be working except for one service. ("coolacid/misp-docker:core-latest") 

I attempted to install MISP by pulling the .yml file via git repository within Portainer and added the variables found in the .env file. After that, I just started the stack. 

Again all services are running except for one and I can't seem to figure out why it is not working. 

Any help is appreciated and let me know if additional information is needed. 

![MISP_Service_Error](https://user-images.githubusercontent.com/26508991/103445419-b3ea6400-4c41-11eb-8e3b-bc33d3f62fef.png)

**Comment by coolacid** [comment]
Is there any logs from when the image is started?

**Comment by PandaLyfe** [comment]
I've downloaded all of the logs for every service except for "MISP-misp" because there were no logs available. 

[MISP_db_logs.txt](https://github.com/coolacid/docker-misp/files/5803393/MISP_db_logs.txt)
[MISP_mail_logs.txt](https://github.com/coolacid/docker-misp/files/5803394/MISP_mail_logs.txt)
[MISP_misp_modules_logs.txt](https://github.com/coolacid/docker-misp/files/5803395/MISP_misp_modules_logs.txt)
[MISP_redis_logs.txt](https://github.com/coolacid/docker-misp/files/5803397/MISP_redis_logs.txt)

Thanks again for the help and let me know if you need any further information.

**Comment by coolacid** [comment]
Yeah, it would need to be the logs from the MISP core container. If it's not even launching the container, I don't know where the problem would be. The only suggestions would be to ensure the some configurations from the docker-compose.yml file exist. Otherwise, it's a problem with Portainer.

**Comment by PandaLyfe** [comment]
Fair enough. Here's some additional information and I hope it helps. 

I attempted to get logs from the MISP_misp service again and it's now showing the following: 

"Error grabbing logs: rpc error: code = Unknown desc = warning: incomplete log stream. some logs could not be retrieved for the following reasons: node 0l0582tvt78dd7n92hcpyr9cs is not available."

Additionally, in Portainer, I clicked on "containers" and located one for MISP_misp. I downloaded the logs and attached it below. 

[MISP_mispl_logs .txt](https://github.com/coolacid/docker-misp/files/5804115/MISP_mispl_logs.txt)

I have OpenCTI running in Portainer, so I think it's running properly. 

Again, thanks for the help and I hope this helps.

**Comment by azzarin** [comment]
I have the same issue. I also have OpenCTI running.

**Comment by GIJosh2687** [comment]
So I see what you mean. I have SSH access to my Portainer stack server. So I went ahead and created that directory that it stated was missing via the inspect ! button on the dashboard.

`sudo mkdir -d /data/compose/2 && cd /data/compose/2`

Then I created each file, stopping and starting the stack to get a new error each time.

`sudo touch files logs server-configs ssl`

I have a new error, I am guessing there is supposed to be some data in that log file.

> No log line matching the '' filter

![image](https://user-images.githubusercontent.com/36141091/180025917-180f5b9e-dd60-4cae-94be-d2152c0f3b35.png)

Anyone have any ideas?
